diff --git a/cmake/modules/FindBLAS.cmake b/cmake/modules/FindBLAS.cmake
index d615983..249daab 100644
--- a/cmake/modules/FindBLAS.cmake
+++ b/cmake/modules/FindBLAS.cmake
@@ -79,9 +79,9 @@ if( BLAS_LIBRARIES MATCHES "[Ii][Mm][Pp][Ll][Ii][Cc][Ii][Tt]" )
 endif()
 
 
-# Check if DGEMM exists in proposed BLAS_LIBRARIES
-check_dgemm_exists( BLAS_LIBRARIES 
-                    BLAS_LINK_OK BLAS_FORTRAN_LOWER BLAS_FORTRAN_UNDERSCORE )
+set( BLAS_LINK_OK TRUE )
+set( BLAS_FORTRAN_LOWER TRUE )
+set( BLAS_FORTRAN_UNDERSCORE TRUE )
 
 
 # If BLAS linkage sucessful, check if it is ILP64/LP64
@@ -95,7 +95,7 @@ if( BLAS_LINK_OK )
     set( _dgemm_name "${_dgemm_name}_" )
   endif()
 
-  check_blas_int( BLAS_LIBRARIES ${_dgemm_name} BLAS_IS_LP64 )
+  set( BLAS_IS_LP64 TRUE )
   if( BLAS_IS_LP64 )
     set( BLAS_lp64_FOUND  TRUE  )
     set( BLAS_ilp64_FOUND FALSE )
diff --git a/cmake/modules/FindLAPACK.cmake b/cmake/modules/FindLAPACK.cmake
index 14b86e4..99a3e7f 100644
--- a/cmake/modules/FindLAPACK.cmake
+++ b/cmake/modules/FindLAPACK.cmake
@@ -119,9 +119,9 @@ endif()
 if( BLAS_HAS_LAPACK )
   set( LAPACK_LINK_OK TRUE )
 else()
-  check_dpstrf_exists( LAPACK_LIBRARIES 
-    LAPACK_LINK_OK LAPACK_FORTRAN_LOWER LAPACK_FORTRAN_UNDERSCORE
-  )
+  set( LAPACK_LINK_OK TRUE )
+  set( LAPACK_FORTRAN_LOWER TRUE )
+  set( LAPACK_FORTRAN_UNDERSCORE TRUE )
 endif()
 
 # If LAPACK linkage sucessful, check if it is ILP64/LP64
@@ -135,7 +135,7 @@ if( LAPACK_LINK_OK )
     set( _dsyev_name "${_dsyev_name}_" )
   endif()
 
-  check_lapack_int( LAPACK_LIBRARIES ${_dsyev_name} LAPACK_IS_LP64 )
+  set( LAPACK_IS_LP64 TRUE )
   if( LAPACK_IS_LP64 )
     set( LAPACK_lp64_FOUND  TRUE  )
     set( LAPACK_ilp64_FOUND FALSE )
